File: app/src/main/java/org/codepond/commitbrowser/common/databinding/ImageUrlAdapter.java
package org.codepond.commitbrowser.common.databinding;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import org.codepond.commitbrowser.R;
import org.codepond.commitbrowser.common.glide.GlideApp;

public class ImageUrlAdapter {
    @BindingAdapter("app:imageUrl")
    public static void loadImage(ImageView imageView, String url) {
        GlideApp.with(imageView.getContext())
                .load(url)
                .fitCenter()
                .placeholder(R.mipmap.ic_launcher) // TODO: Replace placeholder
                .into(imageView);
    }
}

File: app/src/main/java/org/codepond/commitbrowser/common/databinding/ImageUrlAdapter.kt
package org.codepond.commitbrowser.common.databinding

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import org.codepond.commitbrowser.R
import org.codepond.commitbrowser.common.glide.GlideApp

@BindingAdapter("imageUrl")
fun loadImage(imageView: ImageView, url: String) {
    GlideApp.with(imageView.context)
            .load(url)
            .fitCenter()
            .placeholder(R.mipmap.ic_launcher) // TODO: Replace placeholder
            .into(imageView)
}
